kolibri/plugins/coach/assets/src/modules/classSummary/index.js

```diff
@@ -418,13 +435,40 @@ export default {
       state.examLearnerStatusMap = { ...state.examLearnerStatusMap };
       state.contentLearnerStatusMap = { ...state.contentLearnerStatusMap };
     },
+    SET_CLASS_LESSONS_SIZES(state, sizes) {
+      state.lessonsSizes = sizes;
```

Comment:
You need to add an empty `{}` to the default state for each of these maps, otherwise setting the value here will not produce a reactive update. If you don't want to declare this state in advance in the initial state, you'd have to use `Vue.set` here instead of direct property assignment because Vue2.